extensions/wikia/WatchShow/WatchShowHooks.class.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+class WatchShowHooks {
+	public static function onBeforePageDisplay(): bool {
+		\Wikia::addAssetsToOutput( 'watch_show_scss' );
+		\Wikia::addAssetsToOutput( 'watch_show_js' );
+
+		return true;
+	}
+
+	public static function onMercuryWikiVariables( array &$wikiVariables ): bool {
+		global $wgWatchShowURL;
+
+		$wikiVariables['watchShowURL'] = $wgWatchShowURL;
+
+		return true;
+	}
+
+	public static function onGetRailModuleList( Array &$railModuleList ): bool {
```

Comment:
it will fail since this method is void. hook handlers don't need to return anything unless you want to explicitly shortcircuit execution by returning false